lib/google/cloud/gemserver/backend/key.rb

```diff
@@ -58,7 +57,7 @@ class Key
           def self.create_key permissions = nil
             mapped = map_perms permissions
             args = base_args.concat mapped
-            output = capture_stdout { Gemstash::CLI.start args }
+            output = capture_stdout { GemstashServer.start args }
```

Comment:
I think this is a good indication that we've gone in the right direction here. `GemstashServer.start` makes more sense than `Gemstash::CLI.start`.